bin/brew

```diff
@@ -43,11 +56,10 @@ case "$*" in
   --repository|--repo) echo "$HOMEBREW_REPOSITORY"; exit 0 ;;
 esac
 
-if [ "$HOMEBREW_PREFIX" = "/" ] || [ "$HOMEBREW_PREFIX" = "/usr" ]
+if [[ "$HOMEBREW_PREFIX" = "/" ]] || [[ "$HOMEBREW_PREFIX" = "/usr" ]]
```

Comment:
If we can agree on using `[[`, I'd like to embrace its features completely, i.e., use `[[ <cond1> || <cond2> ]]` instead of `[[ <cond1> ]] || [[ <cond2> ]]`. Same for `&&` and moving `!` into the square brackets.